rotkehlchen/exchanges/kraken.py

```diff
@@ -582,90 +582,6 @@ def _query_endpoint_for_period(
             request.update(extra_dict)
         return self.api_query(endpoint, request)
 
-    def query_online_deposits_withdrawals(
-            self,
-            start_ts: Timestamp,
-            end_ts: Timestamp,
-    ) -> list[AssetMovement]:
-        with self.db.conn.read_ctx() as cursor:
-            self.query_kraken_ledgers(cursor, start_ts=start_ts, end_ts=end_ts)
```

Comment:
what now happens to this query?

does it now happen in `query_online_margin_history()`?

Reply:
> does it now happen in query_online_margin_history()?

No, it happens in `query_history_events()`

The flow here is a bit confusing. `query_kraken_ledgers` is called from several different places, and simply loads all events from the kraken api. These events were then being reprocessed later to convert them to AssetMovements, Trades, etc. But now AssetMovements, since they are actually history events, are simply created  in `query_kraken_ledgers` the first time around, and there's no need for any reprocessing.

So where before it was doing something like this to get asset movements into the database: 
`query_deposits_withdrawals` -> `query_online_deposits_withdrawals` -> `query_kraken_ledgers` -> `reprocessing logic in query_online_deposits_withdrawals`

its now simply doing this: 
`query_history_events` -> `query_kraken_ledgers`